site/src/sphinx/advanced-dropwizard-integration.rst

```diff
@@ -0,0 +1,121 @@
+.. _advanced-dropwizard-integration:
+
+Using Armeria with Dropwizard
+=================================
```

Comment:
This is great! Thanks for spending your time on documentation. :heart:

Reply:
Welcome! 📚